README.md

```diff
@@ -42,140 +42,87 @@ the .dom primitives relate to React.
 
 Rendering a very simple DOM structure.
 
-<table width="100%">
-  <tr>
-    <th>React</th>
-    <th>.dom</th>
-  </tr>
-  <tr>
-    <td valign="top">
-<pre lang="javascript">
-ReactDOM.render(
-  React.createElement('div', null, 'Hello world'),
-  document.body
-);
-</pre>
-    </td>
-    <td valign="top">
-<pre lang="javascript">
-R(
-  H('div', 'Hello world'),
-  document.body
-)
-</pre>
-    </td>
-  </tr>
-</table>
+```js
++-----------------------------------+-----------------------------------+
+| React                             | .dom                              |
++===================================+===================================+
+| ReactDOM.render(                  | R(                                |
+|   React.createElement('div', null |   H('div', 'Hello world'),        |
+| , 'Hello world'),                 |   document.body                   |
+|   document.body                   | )                                 |
+| );                                |                                   |
++-----------------------------------+-----------------------------------+
```

Comment:
I really like this, however It makes it a bit difficult to copy-paste the code, so I am kind of torn if this should land or not 🤔 Was your intention to make this friendly on text-editors?